docs/self-managed/zeebe-deployment/security/secure-cluster-communication.md

```diff
@@ -0,0 +1,174 @@
+
+```
+-----BEGIN CERTIFICATE-----
+...
+-----END CERTIFICATE-----
+```
+
+It you wanted to include its signing authority, for example, you would simply append the contents of the authority's public certificate to the end of the certificate chain file.
+
+```
+-----BEGIN CERTIFICATE-----
+...
+-----END CERTIFICATE-----
+-----BEGIN TRUSTED CERTIFICATE-----
+...
+-----END TRUSTED CERTIFICATE-----
+```
+
+While each node will use the default Java trust store to verify incoming certificates (configurable via `javax.net.ssl.trustStore`), which by default uses the system's root certificates, it's recommended to include the complete certificate chain in the file. These will also be used by each node to verify the other nodes' certificates. **More specifically, the certificate chain will be part of the trust store of the node, and will be used to verify other node's certificates**. That will allow you to configure each node with a different leaf certificate sharing the same root certificate (or at least an intermediate authority), as long as they're contained in the chain. Of course, if all nodes use the same certificate, or if you're certain that the certificate is trusted by the root certificates available on each node, then it's sufficient for the file to only contain the leaf certificate.
+
+The private key file should be a PEM private key file, and should be the one during generation of the node's public certificate. Algorithms supported for the private keys are: RSA, DSA, and EC.
+
+:::caution
+
+Note that currently, Zeebe does not support password protected private keys. Since storing the certificates and private keys unencrypted on disk is a security risk, we recommend you use a secret management solution like Vault to inject your certificates in memory at runtime.
+
+:::
+
+## Broker
+
+To configure secure communication for a broker, you will need to configure its `zeebe.broker.network.security` section, which looks like this:
+
+```yaml
+security:
+    # Enables TLS authentication between this gateway and other nodes in the cluster
+    # This setting can also be overridden using the environment variable ZEEBE_BROKER_NETWORK_SECURITY_ENABLED.
+    enabled: false
+
+    # Sets the path to the certificate chain file.
+    # This setting can also be overridden using the environment variable ZEEBE_BROKER_NETWORK_SECURITY_CERTIFICATECHAINPATH.
+    certificateChainPath:
+
+    # Sets the path to the private key file location
+    # This setting can also be overridden using the environment variable ZEEBE_BROKER_NETWORK_SECURITY_PRIVATEKEYPATH.
+    privateKeyPath:
+```
+
+> The `certificateChainPath` and the `privateKeyPath` can be relative to your broker's working directory, or can be absolute paths.
+
+
+## Gateway
+
+To configure secure communication for a standalone gateway with the rest of the cluster, you will need to configure its `zeebe.gateway.cluster.security` section, which looks like this:
```

Comment:
Maybe we need to note that it is different to the grpc tls ? We use between client and gateway?

Reply:
That's already written on the overview page, but it can't hurt to have it here as well, I guess? I'll defer to our editors here @christinaausley - we have two pages, on about client security, and one about cluster security. It's outlined in the security overview page, but I can image users skipping this sometimes and go straight for some link in the sidebar. Do you think we should add a note in both pages, something to the effect of: "If you're looking for client side security, then <link> is the right place", and vice versa.